app/ante.go

```diff
@@ -25,31 +33,50 @@ func NewAnteHandler(options HandlerOptions) (sdk.AnteHandler, error) {
 	if options.SignModeHandler == nil {
 		return nil, sdkerrors.Wrap(sdkerrors.ErrLogic, "sign mode handler is required for ante builder")
 	}
+	if options.OracleKeeper == nil {
+		return nil, sdkerrors.Wrap(sdkerrors.ErrLogic, "oracle keeper is required for AnteHandler")
+	}
 	if options.IBCKeeper == nil {
 		return nil, sdkerrors.Wrap(sdkerrors.ErrLogic, "IBC keeper is required for AnteHandler")
 	}
+	if options.GlobalfeeKeeper == nil {
```

Comment:
Keep in check with the main 0.47 branch that whether we should check stakingkeeper too or not

Reply:
merge from main already.